modules/smart-agent_logstash/variables.tf

```diff
@@ -0,0 +1,375 @@
+  description = "Notification recipients list per severity overridden for events_out_low detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "events_out_low_aggregation_function" {
+  description = "Aggregation function and group by for events_out_low detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
+}
+
+variable "events_out_low_transformation_function" {
+  description = "Transformation function for events_out_low detector (i.e. \".mean(over='5m')\")"
+  type        = string
+  default     = ".mean(over='5m')"
+}
+
+variable "events_out_low_threshold_critical" {
+  description = "Critical threshold for events_out_low detector"
+  type        = number
+  default     = 0
+}
+
+variable "events_out_low_threshold_critical_lasting" {
+  description = "Critical threshold lasting for events_out_low detector"
+  type        = string
+  default     = "5m"
+}
+
+variable "events_out_low_threshold_major" {
+  description = "Major threshold for events_out_low detector"
+  type        = number
+  default     = 10
+}
+
+variable "events_out_low_threshold_major_lasting" {
+  description = "Major threshold lasting for events_out_low detector"
+  type        = string
+  default     = "10m"
+}
+
+# CPU usage
+variable "cpu_usage_percent_tip" {
+  description = "Suggested first course of action or any note useful for incident handling"
+  type        = string
+  default     = ""
+}
+
+variable "cpu_usage_percent_runbook_url" {
+  description = "URL like SignalFx dashboard or wiki page which can help to troubleshoot the incident cause"
+  type        = string
+  default     = ""
+}
+
+variable "cpu_usage_percent_disabled" {
+  description = "Disable all alerting rules for cpu_usage_percent detector"
+  type        = bool
+  default     = null
+}
+
+variable "cpu_usage_percent_disabled_critical" {
+  description = "Disable critical alerting rule for cpu_usage_percent detector"
+  type        = bool
+  default     = null
+}
+
+variable "cpu_usage_percent_disabled_major" {
+  description = "Disable major alerting rule for cpu_usage_percent detector"
+  type        = bool
+  default     = null
+}
+
+variable "cpu_usage_percent_notifications" {
+  description = "Notification recipients list per severity overridden for cpu_usage_percent detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "cpu_usage_percent_aggregation_function" {
+  description = "Aggregation function and group by for cpu_usage_percent detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ""
+}
+
+variable "cpu_usage_percent_transformation_function" {
+  description = "Transformation function for cpu_usage_percent detector (i.e. \".mean(over='5m')\")"
+  type        = string
+  default     = ".mean(over='5m')"
+}
+
+variable "cpu_usage_percent_threshold_critical" {
+  description = "Critical threshold for cpu_usage_percent detector"
+  type        = number
+  default     = 100
```

Comment:
Even if you right used `>=` comparison I am afraid `100` could not be always reached. In doubt I would set `99` or `99.8` if you still want to keep a very high value.